Mathlib/FieldTheory/SimpleExtension.lean

```diff
@@ -0,0 +1,120 @@
+/-
+Copyright (c) 2024 Daniel Weber. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Daniel Weber
+-/
+import Mathlib.FieldTheory.PrimitiveElement
+import Mathlib.RingTheory.Henselian
+import Mathlib.Algebra.Lie.OfAssociative
```

Comment:
Why are these two imports needed in that file? It seems that simple extension does not use Henselian or Lie algebra.

Reply:
I don't know, that's what #min_imports said, but indeed shake complains about it.

Reply:
You can try to remove these two imports and see if it still compiles. (The answer is yes.)

Reply:
Done.